programs/multisig/src/lib.rs

```diff
@@ -217,28 +217,28 @@ pub struct ExecuteTransaction<'info> {
 
 #[account]
 pub struct Multisig {
-    owners: Vec<Pubkey>,
-    threshold: u64,
-    nonce: u8,
-    owner_set_seqno: u32,
+    pub owners: Vec<Pubkey>,
```

Comment:
maybe `pub (crate)` works with the CLI?

Reply:
I think it’s an oversight upstream that these `pub`s were missing, the struct itself is `pub`, and the fields should be public, otherwise nobody else would be able to use `anchor-client` to read the on-chain data. That the CLI client happens to live in the same crate shouldn’t make it a special in this sense.